singularity/init.sh

```diff
@@ -83,7 +84,8 @@ echo '***************Generating Configs***************'
 cd ${containerDataDir}/scripts
 mkdir -p ${containerDataDir}/dpdash/configs \
 && mkdir -p ${containerDataDir}/dpdash/configs/dashboard \
-&& mkdir -p ${containerDataDir}/dpdash/dist
+&& mkdir -p ${containerDataDir}/dpdash/dist \
+&& mkdir -p ${containerDataDir}/dpdash/webpack-build
```

Comment:
Seeing this now, feel free to ignore last comment about `webpack-build` creation